pkg/tsdb/cloudwatch/response_parser.go

```diff
@@ -13,7 +13,8 @@ import (
 )
 
 func (e *cloudWatchExecutor) parseResponse(metricDataOutputs []*cloudwatch.GetMetricDataOutput, queries map[string]*cloudWatchQuery) ([]*cloudwatchResponse, error) {
-	mdr := make(map[string]map[string]*cloudwatch.MetricDataResult)
+	mdrs := make(map[string]map[string]*cloudwatch.MetricDataResult)
```

Comment:
I find helpful to have comment saying what the strings are on map like this, e.g. id -> label -> result.